services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -889,6 +890,9 @@ public static RouteOptions fromUrl(@NonNull URL url) {
         int idx = query.indexOf("=");
         String property = URLDecoder.decode(query.substring(0, idx), UTF_8);
         String value = URLDecoder.decode(query.substring(idx + 1), UTF_8);
+        if (property.equals(ACCESS_TOKEN_URL_PARAM_NAME)) {
+          continue;
+        }
```

Comment:
Do we have a test that covers this already?

Reply:
yep, I added this line because one of tests failed